### DISCUSSION TITLE
App icons stay in the same position

### DISCUSSION THREAD

**Discussion by antonjanrutten** [lawand-dot-io/taskbar#57] [Feature requests]
Right now if you open an app, it moves the icon to the end of the list of apps. This is a bit tricky, as the apps on the taskbar switch locations all the time, making it difficult to track which app is where. The location of these apps can be anywhere as you open more and more apps. I cannot rely on muscle memory to move my cursor to the app I want to open.

I'd like to see app icons stay in one place, and if you open the app, the icon gets highlighted or something to indicate a window is open. This means the app needs to be grouped to support multiple windows which is a separate feature request though.

**Comment by lawand-dot-io** [comment]
@antonjanrutten Thanks for the request. This has been merged into https://github.com/lawand-dot-io/taskbar/discussions/50 since it already includes collapsing grouped app windows into a single icon. Thanks for the request. Feel free to upvote https://github.com/lawand-dot-io/taskbar/discussions/50 to increase the chance that this gets implemented. Also, have you seen the option to group windows which already exists in Taskbar? It doesn't fully achieve what you're asking for, but it reduces the times that a new window is added to the end of the list of windows in Taskbar. Cheers
